install-trainer.sh

```diff
@@ -5,6 +5,7 @@ echo "\nBurro Trainer Installer\n"
 echo "\nFor security and compatibility reasons this installer will not install any system-wide packages.\n"
 echo "\nIf you do not yet have your preferred Tensorflow version, you will have to install it yourself.\n"
 
+sudo apt-get install realpath
```

Comment:
I think that in the training branch we should not install any packages for the user, because there is a small chance their system gets messed up (in Raspberry this is not so important). Perhaps a separate script could be used